kbs/docs/config.md

```diff
@@ -302,6 +342,13 @@ as_addr = "http://127.0.0.1:50004"
 name = "resource"
 type = "LocalFs"
 dir_path = "/opt/confidential-containers/kbs/repository"
+
+[[plugins]]
+name = "nebula-ca"
+nebula_cert_bin_path = "/usr/local/bin/nebula-cert"
+work_dir = "/opt/confidential-containers/kbs/nebula-ca"
+    [plugins.settings]
+    name = "Nebula CA for Trustee KBS"
```

Comment:
Same suggestion as before.

Reply:
Actually, there is no need to have two `nebula-ca` examples, with buit-in and remote CoCo AS. I kept only the one with built-in CoCo AS.